### ISSUE TITLE
Update deno.land module

### ISSUE THREAD

**Issue by piero0920** [ferdousbhai/deno-chat#1]
First at all, thank you so much for the module, it helped me a lot :)

However, the latest version([0.2.0](https://deno.land/x/deno_chat@0.2.0)) of the module on https://deno.land still on commit [#82d2e64](https://github.com/ferdousbhai/deno-chat/commit/82d2e64ffa23e885334662c5da1ec2c4cac158bc).
The `BotProps` doesn't have the `timeout` property.
![Screenshot](https://i.gyazo.com/b11d66c388b1cf47636e91ac6720e5a4.png)

**Comment by ferdousbhai** [comment]
Thank you @piero0920 for your feedback! Sorry, I didn't check my GitHub for the last few weeks. Did you find this library useful? I would be down to build it up more with lot more advanced functionalities.

**Comment by piero0920** [comment]
Yeah, the library is very useful, I been using it ever since I found out about it. 

It would be great to see more advanced functionalities.
